module/icp/io/skein_mod.c

```diff
@@ -231,6 +231,19 @@ skein_mod_init(void)
 
 int
 skein_mod_fini(void) {
+	int ret;
+
+	if (skein_prov_handle != 0) {
+		if ((ret = crypto_unregister_provider(skein_prov_handle)) !=
+		    CRYPTO_SUCCESS) {
```

Comment:
Ahh I see, that explains it.  Well given that I'm OK with either styling.  Either we're not going to be consistent with the other _mod_fini() styling, or with the styling in the rest of the `skein_mod.c` file.  Objection withdrawn. LGTM.